### PR TITLE
feat: add SemVer labels to the Pull Request

### PR DESCRIPTION
**Pull request by KevinDeJong-TomTom**
This feature will add a label (`bump:{version}`) to your Pull Request, based on the valid Conventional Commit messages present in your Pull Request (i.e. `bump:minor`).

This label can be used to either:
- Review whether the right SemVer version will be bumped
- Used by automation (i.e. automatic Release Notes generation)

**NOTE**: This feature requires `write` permissions for both `issues` and `pull-requests` for the used GitHub token.

### CODE REVIEW

File: src/actions/validate.ts [hunk 1]
```diff
@@ -39,7 +40,10 @@ async function run(): Promise<void> {
 
     // Validate each commit against Conventional Commit standard
     const messages = await getMessagesToValidate();
-    await validateMessages(messages, config);
+    const convMessges = await validateMessages(messages, config);
```
Comment (maikelvdh) [inline src/actions/validate.ts:43]:
```suggestion
    const convMessages = await validateMessages(messages, config);
```

Though I think the name with prefix of `conv` is a not so clear one.

File: src/github.ts [hunk 2]
```diff
@@ -264,3 +264,56 @@ export async function getAssociatedPullRequests(
     return [];
   }
 }
+
+/**
+ * Updates the Pull Request (issue) labels to contain the SemVer bump level, in
+ * the format: `bump:<version>`
+ */
+export async function updateSemVerLabel(semverType: SemVerType): Promise<void> {
+  const issue_id = getPullRequestId();
```
Comment (maikelvdh) [inline src/github.ts:273]: Lets remain consistent with naming convention, eg. `issueId`

File: src/github.ts [hunk 3]
```diff
@@ -264,3 +264,56 @@ export async function getAssociatedPullRequests(
...
+  // Retrieve current labels
+  const { data: labels } = await getOctokit().rest.issues.listLabelsOnIssue({
+    owner: OWNER,
+    repo: REPO,
+    issue_number: issue_id,
+  });
+
+  try {
+    // Remove all labels prefixed with "Semver-"
+    for (const lbl of labels) {
```
Comment (maikelvdh) [inline src/github.ts:286]:
```suggestion
    for (const label of labels) {
```

**Review by maikelvdh — CHANGES_REQUESTED** [review]

**Review by maikelvdh — APPROVED** [review]

File: src/actions/validate.ts (hunk 1 above)
Comment (KevinDeJong-TomTom) [reply src/actions/validate.ts]: Renamed variables to `commitMessages` and `compliantMessage` to be more concise on their usage.

**Review by jstmlr — APPROVED** [review]

File: src/github.ts [hunk 4]
```diff
@@ -264,3 +264,56 @@ export async function getAssociatedPullRequests(
...
+
+  // Retrieve current labels
+  const { data: labels } = await getOctokit().rest.issues.listLabelsOnIssue({
+    owner: OWNER,
+    repo: REPO,
+    issue_number: issueId,
+  });
+
+  try {
+    // Remove all labels prefixed with "Semver-"
```
Comment (KevinDeJong-TomTom) [inline src/github.ts:285]:
```suggestion
    // Remove all labels prefixed with "bump:"
```

**Pull request merged**